romancal/tests/test_dms_requirements.py

```diff
@@ -0,0 +1,36 @@
+import json
+import re
+from pathlib import Path
+
+TEST_DIRECTORY = Path(__file__).parent.parent
+TEST_REQUIREMENTS_FILENAME = Path(__file__).parent / "dms_requirement_tests.json"
```

Comment:
Minor nit. You could reuse `TEST_DIRECTORY` here rather than calling `Path(__file__).parent` again.
```
TEST_REQUIREMENTS_FILENAME = TEST_DIRECTORY / "dms_requirement_tests.json"
```